testing/google-cloud-appengine-java8/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
+  <application>vam-veneer</application>
```

Comment:
I could not find a good way to remove the application name from code. So switching to using a placeholder name GCLOUD_PROJECT_ID_HERE